### ISSUE TITLE
oracle_grants / oracle_user - unable to grant read/write on a directory 

### ISSUE THREAD

**Issue by topher-debusk** [oravirt/ansible-oracle-modules#58]
This might be another case of me not knowing what I'm doing, but I can't seem to figure out how to grant read permissions to a directory... I think it's because directory permissions can't be stringed together with other grants.

```
SQL> grant create session, read on directory imp_dir to backup_role;
grant create session, read on directory imp_dir to backup_role
      *
ERROR at line 1:
ORA-00990: missing or invalid privilege
```

Not sure what the right direction is here... I'm no oracle expert, but this is the only grant that I've come across that seems to need to be executed individually. I don't know if including a directories grant section is worth while, or just handling it externally of these modules. I tried doing it in the create_user module as well, but it didn't seem to work either... sorry for bring such a bother - if I was better at Python, I'd try to fix it myself 😄 

Here is my variable for reference:
```
- role: backup_role
    state: present
    grants:
      - create session
      - exp_full_database
      - create job
      - flashback any table
      - read on directory import_dir
      - write on directory import_dir
    privs:
      - select:sys.dba_users
```

**Comment by oravirt** [comment]
I haven't actually tested this myself yet, but I think this falls under object_privs, so try this:
```
- role: backup_role
    state: present
    grants:
      - create session
      - exp_full_database
      - create job
      - flashback any table
    privs:
      - select:sys.dba_users
      - read,write:sys.import_dir
```

**Comment by oravirt** [comment]
No, scratch that. It probably wont work because of the '`directory`' keyword.
I'll have to think about this.

You could use the oracle_sql module to run the actual sql statement for now

**Comment by oravirt** [comment]
I have a solution for this, I will push it later tonight or tomorrow.
I’ll add another list - ‘directory_privs’ which will deal with directories.

**Comment by oravirt** [comment]
Should be fixed with b4b935c065d6cdddf1736050d65f88a674213d9f

**Comment by topher-debusk** [comment]
awesome! you're the man!
